org.gnome.Lollypop.json

```diff
@@ -51,19 +51,6 @@
         "sha256": "5500415b865e8b62775d4742cbb9f37146a50caecfc0e7a6fc0160d3c560fbca"
       }]
     },
-    {
-      "name": "gst-python",
```

Comment:
Any reason for removing this one? :)

Reply:
It didn't compile so I wanted to check with you if we still need it :zipper_mouth_face:

Reply:
We definitely do, see dhttps://gitlab.freedesktop.org/gstreamer/gst-python/-/issues/40
what's the issue you're getting?

Reply:
```
Found ninja-1.10.0 at /usr/bin/ninja
[2/4] Linking target plugin/libgstpython.so
FAILED: plugin/libgstpython.so 
cc  -o plugin/libgstpython.so 'plugin/5e689e2@@gstpython@sha/gstpythonplugin.c.o' -L/app/lib -Wl,--as-needed -Wl,--no-undefined -shared -fPIC -Wl,--start-group -Wl,-soname,libgstpython.so -O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -Wl,-z,relro,-z,now -Wl,--as-needed /usr/lib/x86_64-linux-gnu/libgstreamer-1.0.so /usr/lib/x86_64-linux-gnu/libgobject-2.0.so /usr/lib/x86_64-linux-gnu/libglib-2.0.so /usr/lib/x86_64-linux-gnu/libgstbase-1.0.so -Wl,--export-dynamic /usr/lib/x86_64-linux-gnu/libgmodule-2.0.so -Wl,--end-group -pthread
```
[gst-python.log](https://github.com/flathub/org.gnome.Lollypop/files/5247602/gst-python.log)

Reply:
The issue is caused by gst-python not working properly with python3.8, you will need to backport this https://gitlab.freedesktop.org/gstreamer/gst-python/-/merge_requests/29/diffs for now iirc until the next release of fdo with the new gstreamer release.

Reply:
backported the change. Let's hope it works.